packages/react-aria/package.json

```diff
@@ -42,6 +42,7 @@
     "@react-aria/button": "^3.9.8",
     "@react-aria/calendar": "^3.5.11",
     "@react-aria/checkbox": "^3.14.6",
+    "@react-aria/color": "3.0.0-rc.2",
```

Comment:
assuming this will be auto updated when we release?

Reply:
yeah